SnakeLadder/src/com/snakeladder/model/Driver.java

```diff
@@ -0,0 +1,64 @@
+package com.snakeladder.model;
+
+import java.util.ArrayList;
+
+public class Driver {
+	public static void main(String[] args) throws Exception {
+		int s = 9;
```

Comment:
Please use better variable names

Reply:
I used the same what they provided in the question -

```
Number of snakes (s) followed by s lines each containing 2 numbers denoting the head and tail positions of the snake.
Number of ladders (l) followed by l lines each containing 2 numbers denoting the start and end positions of the ladder.
Number of players (p) followed by p lines each containing a name.
```

Reply:
Ohh. I guess I was too used to CP lingo while creating the problem statement. Trying to get out of it! 😅